### ISSUE TITLE
lint failures do not report which file or line failed

### ISSUE THREAD

**Issue by haarchri** [golangci/golangci-lint-action#814]
### Welcome

- [X] Yes, I understand that the GitHub action repository is not the repository of golangci-lint itself.
- [X] Yes, I've searched similar issues on GitHub and didn't find any.
- [X] Yes, I've included all information below (version, config, etc).

### Description of the problem

The lint github action runs go-lint against the repository.
the lint check is currently failing, but it does not output file names or line numbers, so the failures are very difficult to fix.
any advise ?

![image](https://github.com/golangci/golangci-lint-action/assets/21083497/2c4b4fe5-9175-48b9-84c2-4d74cf105595)

https://github.com/aviatrix/crossplane-provider-aviatrix/actions/runs/5326344384/jobs/9651677439

### Version of golangci-lint

v3

### Version of the GitHub Action

v3

### Workflow file

<details>

```
      # We could run 'make lint' but we prefer this action because it leaves
      # 'annotations' (i.e. it comments on PRs to point out linter violations).
      - name: Lint
        uses: golangci/golangci-lint-action@v3
        with:
          version: ${{ env.GOLANGCI_VERSION }}

```
</details>

### Go version

1.19

### Code example or link to a public repository

https://github.com/aviatrix/crossplane-provider-aviatrix/actions/runs/5326344384/jobs/9651677439

**Comment by gkdr** [comment]
Usually the exact locations of found issues should be reported as annotations, which you can view on the [job summary page](https://github.com/aviatrix/crossplane-provider-aviatrix/actions/runs/5326344384).

However, in your case there are just error messages that say `Cannot open: File exists`, so I believe it's some issue with your setup. The `README` states that this action should run in a separate job, which it doesn't seem to do in your case. Have you tried doing that?

**Comment by codeliger** [comment]
Its way harder to find the annotations than a line number in the config. I wish there was an option for both. Its not working for me either with these settings explicitly enabled:
```
output:
  print-issued-lines: true
  print-linter-name: true
```

**Comment by markusressel** [comment]
Same issue here: https://github.com/markusressel/fan2go/actions/runs/6138849634/job/16656126329?pr=255

**Comment by ldez** [comment]
duplicate of #458
